src/OddDotNet/Span.cs

```diff
@@ -0,0 +1,73 @@
+namespace OddDotNet;
+
+public class Span
+{
+    public required Scope Scope { get; set; }
+    public required string Name { get; set; }
+    public required Dictionary<string, object> Attributes { get; set; }
```

Comment:
Not `required`, no `init` or `set`, and init to empty.

Reply:
Done